lib/java/com/google/android/material/internal/CollapsingTextHelper.java

```diff
@@ -447,6 +449,27 @@ public void apply(Typeface font) {
     recalculate();
   }
 
+  public void setTitleTextEllipsize(int ellipsize) {
```

Comment:
So from the API design perspective, I think we should:

1. Add a public API to CollapsingToolbarLayout so text ellipsize can also be set programmatically.
2. Let the public API take TextUtils.TruncateAt values (or declare new Enums) and pass them here. 
3. Use a conversion method in CollapsingToolbarLayout to convert attribute enums to java enums.

Reply:
@drchen Added public API in CollpasingToolbarLayout. Please review latest changes